app/main.py

```diff
@@ -1,5 +1,71 @@
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        if coords is None:
+            self.coords = [0, 0]
+        else:
+            self.coords = coords
```

Comment:
You can simplify your code
```suggestion
        self.coords = coords or [0, 0]
```